test/configLoading.spec.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+jasmine.getFixtures().fixturesPath = 'base/test/fixtures';
+
+var baseConfig = require('./utils/defaultConfig');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["baseConfig" is defined but never used](https://www.codacy.com/app/laander/booking-js/file/2419776782/issues/source?bid=2969108&fileBranchId=3228106#l5)